typings/jsonld.d.ts

```diff
@@ -1,3 +1,9 @@
 declare module 'jsonld' {
-  export function canonize(doc: any): Promise<string>
+  interface JsonldFactory {
```

Comment:
the function is the factory - the object being returned is a Jsonld instance, not the factory